src/test/javascript/spec/e2e.ts

```diff
@@ -77,4 +91,39 @@ describe('Screenshot Tests', () => {
     expect(await mutationList.isDisplayed()).toBe(false);
     expect(singleMutationView).toExist();
   });
+
+  it('should delete mutation immediately if newly created and not reviewed', async () => {
+    await browser.url(`${BASE_URL}/curation/EMPTYGENE/somatic`);
+
+    // Click to open mutation modal
+    const addMutationButton = await $('button=Add Mutation');
+    await addMutationButton.waitForDisplayed();
+    await addMutationButton.click();
+
+    // Add a new mutation
+    const mutationNameInput = await $('input#add-mutation-modal-input');
+    await mutationNameInput.setValue('V600E');
+    await browser.keys('Enter');
+
+    const addMutationModal = await $('div[id="default-add-mutation-modal"]');
+    await addMutationModal.waitForDisplayed();
+
+    const modalConfirmBtn = await $('button=Confirm');
+    await modalConfirmBtn.click();
```

Comment:
We should share common actions. This one will be used repeatedly I imagine.

Reply:
Will create a test utils